librz/util/graph.c

```diff
@@ -316,3 +316,893 @@ RZ_API void rz_graph_dfs(RzGraph *g, RzGraphVisitor *vis) {
 		free(color);
 	}
 }
+
+#include <rz_util/rz_graph_new.h>
+
+typedef struct rz_graph_list_edge_impl_t {
+	HtUP *adjacent_table; // <ut64, RzList *>
+} RzGraphListEdgeImpl;
+
+typedef struct rz_graph_matrix_edge_impl_t {
+	st64 **adj_matrix;
```

Comment:
```suggestion
	ut8 **adj_matrix;
```

Any reason to make the cells so big? With `ut8` we could already encode 255 edge types.

Reply:
Or `bit0` is set or unset. And the upper bits are some other properties. Edge color for example.